modules/cluster-logging-updating-logging-to-current.adoc

```diff
@@ -0,0 +1,218 @@
+:_content-type: PROCEDURE
+[id="cluster-logging-updating-logging-to_current_{context}"]
+= Updating OpenShift Logging to the current version
```

Comment:
Should `OpenShift` or `OpenShift Logging` be using attributes? (Here and throughout this PR)

Reply:
No, not for Operator names.